runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -3391,9 +3389,9 @@ makeCopyingTROTInduction1Graph(TR::Compilation *c, int32_t ctrl, int32_t pattern
    }
 
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
+
+
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there's no need to change these lines. May I ask you to change them back to the following?

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```